internal/venice-common/src/test/java/com/linkedin/venice/schema/SchemaAdapterTest.java

```diff
@@ -26,7 +26,7 @@ public class SchemaAdapterTest {
   private static final Schema EVOLVED_RECORD_SCHEMA = AvroSchemaParseUtils
       .parseSchemaFromJSONStrictValidation(loadFileAsString("AvroRecordUtilsTest/EvolvedRecordSchema.avsc"));
 
-  private static String loadFileAsString(String fileName) {
+  public static String loadFileAsString(String fileName) {
```

Comment:
Can you please put it in a shared util class if we want to reuse this? Making random classes depend on a function from `SchemaAdapterTest` is not clean